### ISSUE TITLE
Cant connect to director from another device

### ISSUE THREAD

**Issue by yjwong1999** [intel/openfl#364]
Hi @aleksandr-mokrov
I have tried out the OpenFL framework and implemented my own custom application. 
**So far it works great!** I would like to scale up my application, using additional devices for the collaborative training.
However, I couldn't get other devices to connect to the host device that runs the OpenFL Director.

For simplicity's sake, assuming I'm using the tutorial provided in **openfl-tutorials/interactive_api/Tensorflow_MNIST**
I host the director and envoys on **separate devices**. 
Both devices are using **Ubuntu 18**

However, I received the error as shown below:
    ERROR    Failed to report shard info: <_InactiveRpcError of RPC that terminated with:                                                    envoy.py:150
                              status = StatusCode.UNAVAILABLE                                                                                                     
                              details = "failed to connect to all addresses" 

**To Reproduce**
On Device A (Director)
1.          cd openfl-tutorials/interactive_api/Tensorflow_MNIST/director
2.          ./start_director.sh

On Device B (Envoy)
1.          Change the code in "start_envoy.sh" to:
             fx envoy start -n "$ENVOY_NAME" --disable-tls --envoy-config-path "$ENVOY_CONF" -dh "192.168.1.XXX" -dp 50051
             (I replace the "-dh localhost" to my IP address as shown above)
2.          cd openfl-tutorials/interactive_api/Tensorflow_MNIST/envoy
3.          ./start_envoy.sh env_one envoy_config_one.yaml

**Basically, I only change the code in "start_envoy.sh", in which I change the localhost to the IP address**

Did I miss out on any other steps? 
Or is it because I changed the code in "start_envoy.sh" wrongly?
Thank you very much for any suggestions!

**Comment by aleksandr-mokrov** [comment]
Hi, @yjwong1999! Thanks for your feedback!
Have you changed `listen_host` to your IP address in director_config.yaml (inside openfl-tutorials/interactive_api/Tensorflow_MNIST/director)?

**Comment by yjwong1999** [comment]
Hi @aleksandr-mokrov 
I just realised I have not changed the listen_host in director_config.yaml
I can now connect to the director using another device.

Thank you so much for the prompt reply!
Very nice repo and feedback from the OpenFL Team!
